extension/manifest.json

```diff
@@ -67,32 +66,41 @@
     }
   ],
   "background": {
-    "page": "/js/background_page/background_page.htm"
+    "service_worker": "/js/service_worker/background.js",
+    "type": "module"
   },
   "options_ui": {
     "page": "/chrome/settings/index.htm",
     "open_in_tab": true
   },
   "web_accessible_resources": [
-    "/css/webmail.css",
-    "/img/svgs/reply-icon.svg",
-    "/img/svgs/spinner-white-small.svg",
-    "/img/svgs/spinner-green-small.svg",
-    "/img/svgs/unlock.svg",
-    "/img/logo/flowcrypt-logo-19-19.png",
-    "/chrome/elements/compose.htm",
-    "/chrome/elements/pgp_block.htm",
-    "/chrome/elements/setup_dialog.htm",
-    "/chrome/elements/attachment.htm",
-    "/chrome/elements/attachment_preview.htm",
-    "/chrome/elements/passphrase.htm",
-    "/chrome/elements/add_pubkey.htm",
-    "/chrome/elements/pgp_pubkey.htm",
-    "/chrome/elements/backup.htm",
-    "/chrome/elements/verification.htm",
-    "/chrome/elements/shared/footer.htm",
-    "/chrome/elements/oauth2.htm"
+    {
+      "resources": [
+        "/css/webmail.css",
+        "/img/svgs/reply-icon.svg",
+        "/img/svgs/spinner-white-small.svg",
+        "/img/svgs/spinner-green-small.svg",
+        "/img/svgs/unlock.svg",
+        "/img/logo/flowcrypt-logo-19-19.png",
+        "/chrome/elements/compose.htm",
+        "/chrome/elements/pgp_block.htm",
+        "/chrome/elements/setup_dialog.htm",
```

Comment:
Some whitelisted pages here does not actually exists such as:

```bash
"/chrome/elements/setup_dialog.htm",
 "/chrome/elements/setup_dialog.htm",
 "/chrome/elements/verification.htm",
 ```

I think they can be removed.

Reply:
Yeah, it can be indeed removed :+1: